src/rust/bitbox02-rust-c/src/alloc.rs

```diff
@@ -0,0 +1,41 @@
+#[alloc_error_handler]
+#[cfg(not(test))]
+/// Function name is arbitrary.
```

Comment:
Since you are using three slashes this is the documentation for the function. Use two forward slashes if you want to write documentation that isn't documentation.